code/modules/reagents/reagent_dispenser.dm

```diff
@@ -196,32 +196,40 @@
 			log_game("[key_name(Proj.firer)] shot fueltank at [loc.loc.name] ([loc.x],[loc.y],[loc.z]).",ckey=key_name(Proj.firer))
 
 		if(!istype(Proj ,/obj/item/projectile/beam/lastertag) && !istype(Proj ,/obj/item/projectile/beam/practice) && !istype(Proj ,/obj/item/projectile/kinetic))
-			ex_act(2.0)
+			if(is_leaking)
+				if(prob(45))
+					ex_act(1.0)
+				else
+					return
+			else if(prob(35))
+				ex_act(1.0)
+			else
+				is_leaking = TRUE
 
 /obj/structure/reagent_dispensers/fueltank/ex_act(var/severity = 3.0)
 
 	if (QDELETED(src))
 		return
 
 	if (reagents.total_volume > 500)
-		explosion(src.loc,1,2,4)
+		explosion(src.loc,-1,3,2)

```

Comment:
Today in explosioncode fun facts: this change does largely bugger-all if live uses recursive/iterative explosions. Which I'm 95% sure it does.

Reply:
Double-checking the code, this just reduces the effective power from 8 to 5.